### DISCUSSION TITLE
Core10GBaseKR PHY Libero Project?

### DISCUSSION THREAD

**Discussion by bschwan** [Mi-V-Soft-RISC-V/miv-rv32-documentation#18] [Bare Metal Embedded Software] [answered]
Is there a Libero project associated with the core10Gbasekr-example https://github.com/Mi-V-Soft-RISC-V/miv-rv32-bare-metal-examples/tree/main/driver-examples/Core10gBaseKR-PHY/miv-rv32-core10gbasekr-example? If not, could one be added to go with the example?

Thanks in advance.

**Comment by CLappin** [comment, marked as answer]
Hi @bschwan,

We are currently planning the design for the firmware. This will take some time as we will need to test and produce some documentation to accompany it. This design will be released on the  microchip website, but we will update this discussion post when it has been released or if you'd like you can contact technical support. 

Sorry for any inconvenience.

Ciaran
